File: fizz_buzz.js

var readline = require('readline-sync');

var question = readline.question("What range would you like?");

console.log(question);

const answer = Array.from({length: parseInt(question) + 1}, (x,i) => i);

answer.shift();

answer.forEach(function(element) {
  if (element % 3 == 0 && element % 5 == 0) {
    console.log("FizzBuzz");
  } else if (element % 3 == 0){
    console.log("Fizz");
  } else if (element % 5 == 0) {
    console.log("Buzz");
  } else {
    console.log(element);
  }
});

File: fizz_buzz.rb
puts "What range do you want?"

input = gets.chomp.to_i

input.times do |num|
  if num % 3 == 0 && num % 5 == 0
  puts "FizzBuzz"
  elsif num % 3 == 0
    puts "Fizz"
  elsif num % 5 == 0
    puts "Buzz"
  else
    puts num
  end
end
